### PR TITLE
feat: add AVAX, wAVAX

### PR DESCRIPTION
**Pull request by Imti**

### CODE REVIEW

File: src/config/tokens.ts [hunk 1]
```diff
@@ -543,4 +544,52 @@ export const mainnetTokens: TokenMetadataMap = {
...
+  AVAX: {
+    nativeNetwork: 'avalanche',
+    symbol: 'AVAX',
+    name: 'Avalanche',
+    icon: AVAXIcon,
+    iconColors: ['#fff', '#e84142'],
+    decimals: 18,
+    coinGeckoId: 'avax',
+    tokenIdentifier: null,
+    nativeOnly: true,
```
Comment (ErinHales) [inline src/config/tokens.ts:580]: `wrappedAsset: 'wAVAX'`

File: src/utils/config.ts [hunk 2]
```diff
@@ -32,6 +33,7 @@ const rpcs: { [key: string]: string[] } = {
   milkomedatestnet: [VUE_APP_MILKOMEDA_TESTNET_RPC],
   xdai: [VUE_APP_XDAI_RPC],
   evmostestnet: [VUE_APP_EVMOS_TESTNET_RPC],
+  avalanche: [VUE_APP_AVALANCHE_RPC],
```
Comment (ErinHales) [inline src/utils/config.ts:36]: Make sure we have this env variable in netlify before we merge

File: src/config/tokens.ts [hunk 3]
```diff
@@ -543,4 +544,52 @@ export const mainnetTokens: TokenMetadataMap = {
...
+    minAmt: 10,
+  },
+  AVAX: {
+    nativeNetwork: 'avalanche',
+    symbol: 'AVAX',
+    name: 'Avalanche',
+    icon: AVAXIcon,
+    iconColors: ['#fff', '#e84142'],
+    decimals: 18,
+    coinGeckoId: 'avax',
```
Comment (ErinHales) [inline src/config/tokens.ts:578]: this is actually 'avalanche'

File: src/utils/config.ts (hunk 2 above)
Comment (Imti) [reply src/utils/config.ts:36]: Yep, added

File: src/config/tokens.ts [hunk 4]
```diff
@@ -543,4 +544,29 @@ export const mainnetTokens: TokenMetadataMap = {
...
+    coinGeckoId: 'avalanche',
+    tokenIdentifier: null,
+    nativeOnly: true,
+    minAmt: 10,
+    wrappedAsset: 'wAVAX',
+  },
+  wAVAX: {
+    nativeNetwork: 'avalanche',
+    symbol: 'wAVAX',
+    name: 'Avalanche',
```
Comment (ErinHales) [inline src/config/tokens.ts:563]: Should be Wrapped AVAX

**Review by ErinHales — APPROVED** [review]

**Pull request merged**